src/ng/directive/attrs.js

```diff
@@ -144,7 +144,7 @@
  *
  * The correct way to write it:
  * ```html
- * <img ng-srcset="http://www.gravatar.com/avatar/{{hash}} 2x"/>
+ * <img ng-srcset="http://www.gravatar.com/avatar/{{hash}} 2x" alt="Description" />
```

Comment:
I suggest the same change is made to the buggy way above.
Since this example is about `src` vs `ngSrc`, we don't want to distract the reader into thinking `alt` has anything to do with that.